theme/tutorial-sidebar.less

```diff
@@ -973,11 +904,124 @@
     }
 }
 
+/*******************************
+        Media Adjustments
+*******************************/
+
+@media only screen and (min-width: @largestSmallMonitor) {
+    #root.tabTutorial:not(.fullscreensim, .greenscreen) {
+        #editortools {
+            width: calc(100% - @simulatorWidth);
+            left: unset;
+            right: 0;
+        }
+    }
+}
+
+
+@media only screen and (min-width: @largestTabletScreen) and (max-width: @largestSmallMonitor) {
+    #root.tabTutorial:not(.fullscreensim, .greenscreen) {
+        #editortools {
+            width: calc(100% - @simulatorWidthSmall);
+            left: unset;
+            right: 0;
+        }
+    }
+}
+
+/* Desktop Only */
+@media only screen and (min-width: @largestTabletScreen) {
+    #root.tabTutorial:not(.fullscreensim, .greenscreen) {
+        .editor-sidebar:not(.topInstructions) {
+            bottom: 0;
+        }
+
+        .cloudstatusarea .cloudtext {
+            display: none;
+        }
+    }
+    .editor-sidebar:not(.topInstructions) .tutorial-container .tutorial-step-counter {
+        .tutorial-step-label {
+            display: none;
+        }
+    }
+
+    .editor-sidebar:not(.topInstructions) .tutorial-step-label {
+        font-size: 1rem;
+        justify-content: space-between;
+        margin-bottom: 1.5rem;
+    }
+
+    /* Short Deskstop Only */
+    @media (max-height: @tallEditorBreakpoint) {
+        #root.tabTutorial:not(.fullscreensim) {
+            .simulator-container:not(.hidden) {
+                .simPanel {
+                    height: calc(100% - @tutorialControlsOffset);
+                    min-height: @tutorialSimulatorMinHeight;
+                }
+                #boardview {
+                    height: calc(100% - @tutorialBoardviewOffset);
+                    min-height: @tutorialSimulatorMinHeight;
+                }
+                #simulators {
+                    height: 100%;
+                    min-height: @tutorialSimulatorMinHeight;
+                }
+                .simframe {
+                    padding-bottom: unset !important;
+                    height: 100%;
+                    min-height: @tutorialSimulatorMinHeight;
+                }
+            }
+        }
+    }
+
+    .topInstructionsWrapper {
+        .topInstructionsSettings();
```

Comment:
These mixins are really neat! Thanks for introducing them to me.

Reply:
Yeah, very handy! I'm not sure I'm using them 100% correctly, but it seems to work? So I'm happy with it :)